addons/medical/functions/fnc_setUnconscious.sqf

```diff
@@ -19,10 +19,13 @@
 
 #include "script_component.hpp"
 
-#define DEFAULT_DELAY (round(random(10)+5))
+#define DEFAULT_DELAY   (round(random(10)+5))
 
-private ["_animState", "_originalPos", "_startingTime", "_isDead"];
-params ["_unit", ["_set", true], ["_minWaitingTime", DEFAULT_DELAY], ["_force", false]];
+private ["_unit", "_set", "_animState", "_originalPos", "_startingTime","_minWaitingTime", "_force", "_isDead"];
+_unit = _this select 0;
+_set = if (count _this > 1) then {_this select 1} else {true};
+_minWaitingTime = if (count _this > 2) then {_this select 2} else {DEFAULT_DELAY};
+_force = if (count _this > 3) then {_this select 3} else {false};
```

Comment:
Technically this is not the same, because

count [0,nil,1]
-> 3

But I doubt we'd call something like this.